README.md

```diff
@@ -32,6 +35,9 @@ The `"vidarr"` section describes all the Vidarr instances that should be used
 as file sources. The keys are the _internal name_ of that Vidarr instance and
 the value is the URL of that instance.
 
+The `"ignore"` section contains all the LIMS data sources which are present 
+in the vidarr instances but should NOT be merged.
```

Comment:
What do you think of:

The `"ignore"` section contains all the LIMS provider names which are present 
in the vidarr instances' external keys but should NOT be merged when building
file provenance. If a Vidarr workflow run contains a single external key with data
from one of these `ignore` providers, the entire workflow run will be excluded.